.circleci/config.yml

```diff
@@ -199,6 +230,20 @@ workflows:
             tags:
               only: /^v[0-9]+(\.[0-9]+)*$/
 
+      - test_deprecation_warnings:
+          matrix:
+            parameters:
+              version:
+                - "3.9"
```

Comment:
I think this should be the latest aka most recent supported version. Otherwise issues are detected when it's already too late.